### DISCUSSION TITLE
Like and Boost counts

### DISCUSSION THREAD

**Discussion by hardillb** [Automattic/wordpress-activitypub#1116] [General]
I thought I saw a comment in a recent set of release notes about showing Like and Boost counts for a post, but I'm not seeing this on my posts.

Could this be because I am using the old Twenty Fifteen theme? And if so is there an edit I can make to the theme include this?

Thanks

**Comment by pfefferle** [comment]
The first version is only for FSE themes.

**Comment by IAmDarthMole** [comment]
While it is an extra step you can manually add the "Fediverse Reactions" block to the end of your posts if you are using a none FSE theme (like me).
